project/src/main/java/com/demo/App.java

```diff
@@ -0,0 +1,17 @@
+package com.demo;
+
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+
+public class App 
+{
```

Comment:
Apply formatter to all classes. Current format is not compliant with Java code convention